src/parallel_bush.c

```diff
@@ -180,6 +180,7 @@ void reconstructMerges_par(int origin, network_type *network, bushes_type *bushe
                        "incoming contributing links", i, origin);
         if (numApproaches == 1) { /* No merge */
             bushes->pred[origin][i] = lastApproach;
+            if (bushes->flow_par[origin][lastApproach] == NEW_LINK) bushes->flow_par[origin][lastApproach] = 0;
```

Comment:
What were the values before? Just random junk numbers from initialization?

Reply:
they were just left at -1. However, it caused additional links to be added into the bush merge structure creating cycles

Reply:
This bug seemed to appear in the extreme case of NCTCOG